apps/system/test/marionette/pinning_the_web_test.js

```diff
@@ -100,4 +99,36 @@ marionette('Pinning the Web', function() {
     assert(classes.indexOf('collapsible') < 0);
   });
 
+  suite.skip('Unpin site', function() {
```

Comment:
I think the pin the web features of the doorhanger are likely to completely go away in the next release where there will only be the Tracking Protection. I kept them just in case but I wouldn't run them on CI.